src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -29,25 +29,25 @@ def _reindex_lake_to_link_id(target_df, crosswalk):
 
     # evaluate intersection of lake ids and target_df index values
     # i.e. what are the index positions of lake ids that need replacing?
-    lakeids = np.fromiter(crosswalk.keys(), dtype=int)
+    lakeids = np.fromiter(crosswalk.keys(), dtype = int)
     idxs = target_df.index.to_numpy()
     lake_index_intersect = np.intersect1d(
         idxs,
         lakeids,
-        return_indices=True
+        return_indices = True
     )
 
     # replace lake ids with link IDs in the target_df index array
-    linkids = np.fromiter(crosswalk.values(), dtype=int)
+    linkids = np.fromiter(crosswalk.values(), dtype = int)
     idxs[lake_index_intersect[1]] = linkids[lake_index_intersect[2]]
 
     # (re) set the target_df index
-    target_df.set_index(idxs, inplace=True)
+    target_df.set_index(idxs, inplace = True)
 
     return target_df
 
 
-def _parquet_output_format_converter(df, start_datetime, dt, configuration):
+def _parquet_output_format_converter(df, start_datetime, dt, configuration, prefix_ids):
```

Comment:
We can drop most of these format only changes to keep the PR super clean.

Reply:
I have reverted the format changes.